MantidPlot/src/Spectrogram.h

```diff
@@ -220,6 +226,8 @@ class Spectrogram: public QObject, public QwtPlotSpectrogram, public MantidQt::A
   std::vector<unsigned char> mScaledValues;
   /// boolean flag to indicate intensity changed
   bool m_bIntensityChanged;
+  /// Flag for whether we autoscale color bar
+  bool d_color_map_autoscale;
```

Comment:
Coding standards for Mantid have all member variables pre-fixed with `m_` this is the only aspect of Hungarian format we follow.